tests/contrib/django_util/test_views.py

```diff
@@ -180,9 +181,10 @@ def test_callback_works(self, pickle):
         self.assertEqual(
             response.status_code, django.http.HttpResponseRedirect.status_code)
         self.assertEqual(response['Location'], self.RETURN_URL)
+        jsonpickle_mock.decode.assert_called_once_with(pickled_flow)
```

Comment:
Was `encode` called (other than your `jsonpickle_mock.encode(flow)` above)? Also, what is the point of that? Wouldn't `pickled_flow = object()` work just fine?